packages/create-sitecore-jss/src/common/Initializer.ts

```diff
@@ -1,11 +1,22 @@
 import { BaseArgs } from './args/base';
 
-export interface InitializerResults {
+export type InitializerResults = {
   appName: string;
   initializers?: string[];
   nextSteps?: string[];
-}
-export interface Initializer {
+};
+
+/**
+ * Initializer base type
+ */
+export type Initializer = {
```

Comment:
For class interfaces, I think we should stick to `interface` (common in OO languages,[ TS docs](https://www.typescriptlang.org/docs/handbook/2/classes.html#implements-clauses) seem to prefer this too).